README.md

```diff
@@ -1,3 +1,3 @@
-# GraalVM Dev Build Downloads
+# GraalVM Dev Build Downloads [![Latest dev release](https://img.shields.io/github/v/release/graalvm/graalvm-ce-dev-builds?color=brightgreen&label=latest%20release)](https://github.com/graalvm/graalvm-ce-builds/releases/latest)
```

Comment:
Link https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest is incorrect
should be https://github.com/graalvm/graalvm-ce-dev-builds/releases/latest

Reply:
Nice catch @ezzarghili you are right, thanks.

Reply:
Done